core/src/com/unciv/ui/worldscreen/unit/UnitActions.kt

```diff
@@ -306,11 +307,28 @@ object UnitActions {
 
     fun getPillageAction(unit: MapUnit): UnitAction? {
         val tile = unit.currentTile
-        if (unit.isCivilian() || tile.improvement == null || tile.getOwner() == unit.civInfo) return null
+        println(tile.roadStatus)
+        if (unit.isCivilian() || (tile.improvement == null && tile.roadStatus == RoadStatus.None) || tile.getOwner() == unit.civInfo) return null
 
         return UnitAction(UnitActionType.Pillage,
                 action = {
-                    tile.getOwner()?.addNotification("An enemy [${unit.baseUnit.name}] has pillaged our [${tile.improvement}]", tile.position, "ImprovementIcons/${tile.improvement!!}", NotificationIcon.War, unit.baseUnit.name)
+                    if (tile.improvement == null) {
+                        tile.getOwner()?.addNotification(
+                            "An enemy [${unit.baseUnit.name}] has pillaged our [${tile.roadStatus.name}]",
+                            tile.position,
+                            "ImprovementIcons/${tile.roadStatus.name}",
+                            NotificationIcon.War,
+                            unit.baseUnit.name
+                        )
+                    } else {
+                        tile.getOwner()?.addNotification(
+                            "An enemy [${unit.baseUnit.name}] has pillaged our [${tile.improvement}]",
+                            tile.position,
+                            "ImprovementIcons/${tile.improvement!!}",
+                            NotificationIcon.War,
+                            unit.baseUnit.name
+                        )
+                    }
```

Comment:
This looks like code duplication; why not add a variable containing the name of the thing that is pillaged and use that instead of adding the same code twice with just one differing variable?